docs/content/Deployment/Platform-Kubernetes.md

```diff
@@ -0,0 +1,516 @@
+    spec:
+      containers:
+        - env:
+            - name: ALLOW_EMPTY_PASSWORD
+              value: 'yes'
+          image: bitnami/redis:latest
+          name: redis
+          ports:
+            - containerPort: 6379
+          resources: {}
+      restartPolicy: Always
+status: {}
+```
+
+And a corresponding service for the Redis deployment in a file named
+`redis-service.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: redis
+  name: redis
+spec:
+  ports:
+    - name: '6379'
+      port: 6379
+      targetPort: 6379
+  selector:
+    io.kompose.service: redis
+status:
+  loadBalancer: {}
+```
+
+Next, create a file called `cube-api-deployment.yaml` with the following
+contents:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: cube-api
+  name: cube-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: cube-api
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        io.kompose.service: cube-api
+    spec:
+      containers:
+        - env:
+            - name: CUBEJS_API_SECRET
+              value: secret
+            - name: CUBEJS_CUBESTORE_HOST
+              value: 'cubestore-router'
+            - name: CUBEJS_CUBESTORE_PORT
+              value: '3030'
+            - name: CUBEJS_DB_CLICKHOUSE_READONLY
+              value: 'true'
+            - name: CUBEJS_DB_HOST
+              value: github.demo.trial.altinity.cloud
+            - name: CUBEJS_DB_NAME
+              value: default
+            - name: CUBEJS_DB_PASS
+              value: demo
+            - name: CUBEJS_DB_PORT
+              value: '8443'
+            - name: CUBEJS_DB_SSL
+              value: 'true'
+            - name: CUBEJS_DB_TYPE
+              value: clickhouse
+            - name: CUBEJS_DB_USER
+              value: demo
+            - name: CUBEJS_DEV_MODE
+              value: 'true'
+            - name: CUBEJS_EXTERNAL_DEFAULT
+              value: 'true'
+            - name: CUBEJS_REDIS_URL
+              value: redis://redis:6379
+          image: cubejs/cube:v0.28.14
+          name: cube-api
+          ports:
+            - containerPort: 3000
+            - containerPort: 4000
+          resources: {}
+          volumeMounts:
+            - mountPath: /cube/conf
+              name: cube-api-hostpath0
+      restartPolicy: Always
+      volumes:
+        - hostPath:
+            path: /home/docker/conf
+          name: cube-api-hostpath0
+status: {}
+```
+
+We'll also create a corresponding service for this deployment in a file called
+`cube-api-service.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: cube-api
+  name: cube-api
+spec:
+  ports:
+    - name: '3000'
+      port: 3000
+      targetPort: 3000
+    - name: '4000'
+      port: 4000
+      targetPort: 4000
+  externalIPs:
+    - 192.168.99.103
```

Comment:
We should either remove or comment out the `externalIp`.

We could comment it out and add a comment stating that it is only used for Minikube.

Like this:

```suggestion
  # THIS IS ONLY USED FOR EXPOSING AN IP WHEN USING MINIKUBE
  # externalIPs:
  #   - xxx.zzz.yyy.www
```